### PR TITLE
Adding ams0 as a milestone maintainer

### PR DESCRIPTION
**Pull request by ams0**
/cc @kubernetes/release-team-leads
/cc @erismaster

### CODE REVIEW

**Comment by erismaster** [comment]
/lgtm

Welcome!!

**Comment by soniasingla** [comment]
#2426 is opened for access, once that PR has been closed, all the checks will get passed!

File: config/kubernetes/sig-release/teams.yaml [hunk 1]
```diff
@@ -31,6 +31,7 @@ teams:
     - ahg-g # Scheduling
     - alejandrox1 # Testing
     - ameukam # Release Manager Associate
+    - ams0 # Bug Triage Shadow for Kubernetes 1.21
```
Comment (palnabarun) [inline config/kubernetes/sig-release/teams.yaml:34]:
```suggestion
    - ams0 # 1.21 Bug Triage Shadow
```

**Review by palnabarun — CHANGES_REQUESTED** [review]
@erismaster -- do you want to possibly add all the Bug Triage shadow in a single PR?

Also, the previous Bug Triage team needs to be removed if they don't need access.

**Comment by erismaster** [comment]
@palnabarun good call - I’ll get this reconciled tomorrow! We can probably close this one and #2424

**Comment by palnabarun** [comment]
@erismaster -- Awesome! Thank you! :)

**Comment by palnabarun** [comment]
@erismaster -- #2424 is fine. Daniel is added to the Kubernetes org now.

**Comment by palnabarun** [comment]
/close

(ref: https://github.com/kubernetes/org/pull/2425#issuecomment-762008384)

**Pull request closed without merging**